Dockerfile

```diff
@@ -18,6 +18,9 @@ RUN npm run build
 FROM alpine:3.7
 LABEL maintainer="Hypothes.is Project and contributors"
 
+# Expose the default port.
+EXPOSE 5000
```

Comment:
This command has never changed in the lifetime of the Dockerfile AFAIK and `CMD` changes very rarely. My suggestion with regards to placement is that we should just do whatever is most idiomatic - whether that's near the top or bottom of the file.

Reply:
This[https://runnable.com/blog/9-common-dockerfile-mistakes#annotations:oFgUVg-EEemcvavOf28b7Q] actually says to place them at the end because they are light enough it doesn't really matter and if for some reason they change it's quick to rebuild. It seems a little counter intuitive to me but that seems to be the convention in the docker doc examples too although I haven't found a place where it specifically mentions the order. I suppose it's true, they are small enough it's prob not a big deal to rebuild them every time.